include/kafka/AdminClient.h

```diff
@@ -74,8 +79,18 @@ class AdminClient: public KafkaClient
      */
     Admin::ListTopicsResult   listTopics(std::chrono::milliseconds timeout = std::chrono::milliseconds(DEFAULT_COMMAND_TIMEOUT_MS));
 
+    /**
+     * Delete records before given offset for a partition
+     * @param topicPartitionOffsets a batch of offsets for partitions
+     * @param timeout
+     * @return
+     */
+    Admin::DeleteRecordsResult deleteRecords(const TopicPartitionOffsets& tpos,
```

Comment:
`topicPartitionOffsets` is a better name than `tpos`, --  it's for the declaration and should be easy to understand.

Reply:
Sure. Actually, I was not certain about this naming when writing this part, and found `KafkaConsumer.h` uses such abbreviation so I follow the naming there. Now I change it (and the following usages as well) to `topicPartitionOffsets`.